feat: sidebar component has been added but not finished yet

ayanwumi101 opened the pull request:
I have created the sidebar menu and i'm still working on it, i have made some little changes to the styles

ayanwumi101 committed:
feat: sidebar component has been added but not finished yet

LloydRivers reviewed (changes requested):
When you do a pull request, all the changes should have already been done. 

Things left:
add the toggle dark mode button along with the 2 svgs. You can find both of them in the assets folder.

Add the "hide sidebar" button and svg.

I checked it on mobile and it looks like this. Maybe add a high z index for now until we can get more of the interface in.

You have a navContainer doing nothing. I removed it and the page looked identical.


<img width="458" alt="branch review" src="https://user-images.githubusercontent.com/86473036/183988089-3ec8ff4e-fdfc-499b-a555-2de50c539710.png">

LloydRivers committed:
Merge branch 'main' into pr/48

LloydRivers committed:
prettier formatted the code

LloydRivers committed:
merge main into branch / resolve conflicts

LloydRivers committed:
toggle theme

I renamed the names of the icons to match the figma drawing. Added margin bottom to the logo.

Created a reusable component for the nav list item

LloydRivers committed:
add hover to list items

LloydRivers committed:
create component

LloydRivers committed:
fix syling

LloydRivers committed:
export toggle switch

LloydRivers committed:
create the toggle and remove button

LloydRivers committed:
toggle styling

LloydRivers committed:
create toggle component

MorganBare reviewed (approved):
I don't have much to say. It looks great and the toggle is working perfectly so let's move on.
